mmf/utils/build.py

```diff
@@ -165,6 +186,17 @@ def build_dataloader_and_sampler(
     if not isinstance(dataset_instance, torch.utils.data.IterableDataset):
         other_args = _add_extra_args_for_dataloader(dataset_instance, other_args)
 
+    if is_xla():
+        dataset_type = dataset_instance.dataset_type
```

Comment:
`dataset_type` is not used.

Reply:
I think it's a left over from some intermediate changes. I will remove it.